ui/core-react/src/core-react/utils/IconWebComponent.ts

```diff
@@ -13,6 +13,101 @@ import { Logger } from "@itwin/core-bentley";
 import { UiCore } from "../UiCore";
 import DOMPurify, * as DOMPurifyNS from "dompurify";
 
+/**
+ * Completed SVG cache
+ * (If we get it once, we use it from there afterwards)
+ */
+const svgCache = new Map<string, HTMLElement>();
```

Comment:
Do we really need this map? It should be enough to track the promises which would eliminate some conditionals.

Reply:
As I mention in description, it reduce a bit more the time it takes to show on subsequent display, but I'll remove in for code clarity, I doubt that people will notice `4ms` delay and I suppose keeping the function scope in memory is small too compared to the SVG content anyway.